docs/observability/aws/index.md

```diff
@@ -8,7 +8,19 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src='https://upload.wikimedia.org/wikipedia/commons/9/93/Amazon_Web_Services_Logo.svg' alt="Thumbnail icon" width="65"/>
 
-This feature is available in the following account plans.
+You can access the AWS observability solution through a centrlized page by clicking on the **Home** button and selecting the **AWS Observability** tab from the top menu bar. This page is designed to provide easy navigation to all features and functions of the AWS Observability solution in one place. From the hub, you can navigate to the dashboards, monitors (for alerting), and pre-baked log searches for troubleshooting that come with the AWS Observability solution. 
+
+The hub has the following pages:
+
+* **Explore your AWS Environment**. Provides you access to dashboards that help you monitor your AWS environment.
+* **Troubleshoot with Logs**. Provides log searches to help you get started troubleshooting issues within your AWS environment.
+* **View and Configure Alerts**. Provides you access to monitors that will alert you about issues within your AWS environment.
+* **Explore Intelligent Benchmark Dashboards**. Provides you access to our global benchmark dashboards to help you compare your AWS environment with Sumo benchmarks and identify issues.
+* **Invite your teammates**. Allows you to quickly add other users to your Sumo Logic account. 
+
+![AWS Observability Hub.png](/img/observability/aws_observability_hub_page.jpg)
+
+AWS Observability Solution feature is available in the following account plans.
```

Comment:
Is "solution" meant to be lowercase or capitalized? I don't call how it's branded in product.

```suggestion
The AWS Observability Solution feature is available in the following account plans.
```